decisions/wc_class_decisions.txt

```diff
@@ -293,6 +293,7 @@ decisions = {
 			is_in_society = no
 			religion = kaldorei_religion
 			is_female = yes
+			reproduces_asexually = no
```

Comment:
This line is confusing. I know, that it includes only ancients, but name of scripted trigger...

Reply:
I want to use it for treants later and other plant races.

Reply:
Then rename scripted trigger, because for example harpies may reproduce asexually.

Reply:
Done